From 0c56db4610768b1962d35a2927b53b9d51d9d8b8 Mon Sep 17 00:00:00 2001
From: Randy Dunlap <rdunlap@infradead.org>
Date: Wed, 5 Aug 2026 22:56:34 -0700
Subject: [PATCH 13292/15333] doc tools: fix 'path' typos

Correct "patch" or "patches" to "path" or "paths" in comments.

Signed-off-by: Randy Dunlap <rdunlap@infradead.org>
Signed-off-by: Jonathan Corbet <corbet@lwn.net>
Message-ID: <20260806055634.1293294-1-rdunlap@infradead.org>
---
 tools/docs/documentation-file-ref-check | 2 +-
 tools/docs/sphinx-build-wrapper         | 4 ++--
 2 files changed, 3 insertions(+), 3 deletions(-)

diff --git a/tools/docs/documentation-file-ref-check b/tools/docs/documentation-file-ref-check
index 0cad42f6943b..93b53f3a5567 100755
--- a/tools/docs/documentation-file-ref-check
+++ b/tools/docs/documentation-file-ref-check
@@ -143,7 +143,7 @@ while (<IN>) {
 		# Check if exists, evaluating wildcards
 		next if (grep -e, glob("$ref $fulref"));
 
-		# Accept relative Documentation patches for tools/
+		# Accept relative Documentation paths for tools/
 		if ($f =~ m/tools/) {
 			my $path = $f;
 			$path =~ s,(.*)/.*,$1,;
diff --git a/tools/docs/sphinx-build-wrapper b/tools/docs/sphinx-build-wrapper
index ec83583e229a..6f1163333a47 100755
--- a/tools/docs/sphinx-build-wrapper
+++ b/tools/docs/sphinx-build-wrapper
@@ -99,9 +99,9 @@ class SphinxBuilder:
 
     def get_path(self, path, use_cwd=False, abs_path=False):
         """
-        Ancillary routine to handle patches the right way, as shell does.
+        Ancillary routine to handle paths the right way, as shell does.
 
-        It first expands "~" and "~user". Then, if patch is not absolute,
+        It first expands "~" and "~user". Then, if path is not absolute,
         join self.srctree. Finally, if requested, convert to abspath.
         """
 
-- 
2.55.0

